env/Dockerfile

```diff
@@ -3,28 +3,32 @@
 ARG BASE_IMAGE=ubuntu:bionic
```

Comment:
Possibly worth noting in the commit message text: 18.04 ansible is new enough not to require installing a newer one any more.